neon-processor/src/main/java/com/zing/zalo/neon2/internal/NeonConfig.java

```diff
@@ -0,0 +1,10 @@
+package com.zing.zalo.neon2.internal;
+
+/**
+ * Created by Tien Loc Bui on 17/09/2019.
+ */
+public final class NeonConfig {
```

Comment:
protected?

Reply:
I'm sorry, but what does it mean?

Reply:
Should we change visibility to protected?